test/controllers/staff_dashboard_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class StaffDashboardControllerTest < ActionController::TestCase
+=begin
```

Comment:
That sounds like a bug then, or maybe the response is `:found` but the path is right?